crates/net/network/src/peers/manager.rs

```diff
@@ -759,9 +759,7 @@ impl PeersManager {
                 })
             }
 
-            if self.refill_slots_interval.poll_tick(cx).is_ready() {
-                // this ensures the manager will be polled periodically, see [Interval::poll_tick]
-                let _ = self.refill_slots_interval.poll_tick(cx);
```

Comment:
why is this not necessary any more?

Reply:
replaced with loop, the proper way of doing this